Stresser/main.cpp

```diff
@@ -13,17 +13,21 @@
 #include "../Utils/StandardThread.h"
 #include "../Utils/WindowsEvent.h"
 #include "../Utils/TimeUtils.h"
+#include "../Utils/EventsNames.h"
 
 #include "nlohmann/json.hpp"
 
+#include "ArtifactFactory.h"
+
 #include <iostream>
 
 #include <Windows.h>
 
 using Json = nlohmann::json;
 
 // Used to signal events in the main thread:
-WindowsEvent g_shutdownEvent(L"StopEvent");
+WindowsEvent g_shutdownEvent(STOP_STRESSER);
+WindowsEvent g_stopPolicy(STOP_POLICY);
```

Comment:
Why we need this new event ?
There is any use case which we are going to stop checking policy updates, if it is not the case of application "shutdown"
?

Reply:
the event is about stoping the current policy

Reply:
I'm understand, there is situation where we will stop fetching policy ?

Reply:
Change name to: ":PolicyChange" event